.github/workflows/android.yml

```diff
@@ -75,19 +65,28 @@ jobs:
           SENTRYRC: ${{ secrets.sentryrc_file }}
         run: |
           env_name="${{ env.ENV }}"
+          if [[ "${{github.ref_name }}" == "next"]]; then
+           vercel_env_name="production"
+           vercel pull --yes --environment=production --token=${{ secrets.VERCEL_TOKEN }}
+          else 
+            vercel_env_name="preview"
+            vercel pull --yes --environment=$vercel_env_name --git-branch=${{ github.ref_name }} --token=${{ secrets.VERCEL_TOKEN }}
+          fi
+          cat .vercel/.env.$vercel_env_name.local >> .env.$env_name
           echo $env_name
           cat .env.$env_name
-          echo "adding secrets to .env.$env_name file: ${{ env.SECRET_NAME }}"
+          echo "adding secrets to .env.$env_name file"
           echo "$SENTRYRC" > android/sentry.properties
-          echo "${{ secrets[env.SECRET_NAME] }}" >> .env.$env_name
           echo "REACT_APP_CODE_PUSH_KEY=${{ env.APPCENTER_CODEPUSH_TOKEN }}"  >> .env.$env_name
+          # echo "REACT_APP_CODE_PUSH_KEY=$(grep -Po '(?<=^APPCENTER_CODEPUSH_TOKEN=)\S+' .env.$env_name)" >> .env.$env_name
```

Comment:
@johnsmith-gooddollar @sirpy 
The comment is the solution when we want to keep REACT_APP_CODE_PUSH
but the only place its used is in config.js for: codePushDeploymentKey
so we could also just set this directly to APPCENTER_CODE_PUSH_TOKEN and don't need this change in android.yml

Reply:
or is there a different reason why we use REACT_APP_CODE_PUSH?

Reply:
done